README.md

```diff
@@ -48,6 +52,13 @@ List of supported problem daemons:
 | [KernelMonitor](https://github.com/kubernetes/node-problem-detector/tree/master/pkg/kernelmonitor) | KernelDeadlock | A problem daemon monitors kernel log and reports problem according to predefined rules. |
 
 # Usage
+## Flags
+* `-apiserver-override`: An URI parameter used to customize how node-problem-detector
+connects the apiserver. The format is the same with the
+[`source`](https://github.com/kubernetes/heapster/blob/master/docs/source-configuration.md#kubernetes)
+flag of [Heapster](https://github.com/kubernetes/heapster):
+`http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false&userServiceAccount=false&auth=&insecure=`.
```

Comment:
Use "```" instead of "`".
See the difference:
```
http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false&userServiceAccount=false&auth=&insecure=
```
and
`http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false&userServiceAccount=false&auth=&insecure=`